app/uk/gov/hmrc/hmrcemailrenderer/templates/tdq/tdqHeaderCompliancePartiallyCompliantValidConnectionMethod.scala.txt

```diff
@@ -33,11 +33,11 @@ These are the headers where errors and/or warnings have been found:
 @if(headersValidation.errors.nonEmpty) {
   errors
 @for(error <- headersValidation.errors) {
-    @error.message (in @error.percentage% of requests) }}
+    @error.message (found in @if(error.percentage == 0){fewer than 1%} else {@error.percentage%} of your requests) }}
```

Comment:
This could also be method (+ a simple unit test) on the `Problem` class
As you wish though